qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -95,10 +95,9 @@ CEPH_ID=${CEPH_ID:-mirror}
 RBD_IMAGE_FEATURES=${RBD_IMAGE_FEATURES:-layering,exclusive-lock,journaling}
 MIRROR_USER_ID_PREFIX=${MIRROR_USER_ID_PREFIX:-${CEPH_ID}.}
 MIRROR_POOL_MODE=${MIRROR_POOL_MODE:-pool}
-MIRROR_IMAGE_MODE=${MIRROR_IMAGE_MODE:-journal}
+RBD_MIRROR_MODE=${RBD_MIRROR_MODE:-journal}
```

Comment:
Nit: Swap this line with the previous line so that `MIRROR_POOL_MODE` initial assignment and potential adjustment are together:
```
RBD_MIRROR_MODE=${RBD_MIRROR_MODE:-journal}
MIRROR_POOL_MODE=${MIRROR_POOL_MODE:-pool}
if [ "${RBD_MIRROR_MODE}" = "snapshot" ]; then
  MIRROR_POOL_MODE=image
fi
```